def raise_error(error_message):
    pass


